eWonConnector-gateway/src/main/java/com/hms_networks/SyncManager.java

```diff
@@ -232,41 +232,50 @@ public DeletionResponse process(TagPath t) {
         // Verify integrity of tag history storage information if tag history enabled
         if (historyEnabled && StringUtils.isBlank(tagHistoryStore)) {
             logger.warn(
-                    "History sync is enable, but no history provider has been specified for storage."
-                            + "No data will be stored.");
+                    "History synchronization is enabled, but no history provider " +
+                        "has been specified for storage. No data will be stored.");
         }
 
         // Output warning if tag name check is disabled
         if (tagNameCheckDisabled) {
-            logger.warn("Tag name checking is disabled. Tags with unsupported characters may not " +
-                "work properly.");
+            logger.warn("Strict checking for allowed tag name characters is " +
+                    "disabled. Tags with unsupported characters may not work properly.");
         }
 
         // Load and store synchronization data, and create it if necessary
         syncData = gatewayContext.getPersistenceInterface().find(EwonSyncData.META, 1L);
         if (syncData == null) {
-            logger.info("Ewon sync data not found, initializing.");
+            logger.info("Existing Ewon synchronization data not found, " +
+                "creating new data...");
```

Comment:
What does "Creating new data" mean in this context

Reply:
changed to "creating new data store"